glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/minishift/minishift
 import:
 - package: github.com/docker/machine
-  version: 91e368eb7422665278b1d5665139ee0f9980b588
+  version: 0.9.0
```

Comment:
Thanks :-). Eventually I would like to switch as many as dependencies to using this form of version.